### DISCUSSION TITLE
Query reg: Normalized signal - CoveragePlot

### DISCUSSION THREAD

**Discussion by jarchana09** [stuart-lab/signac#1172] [Q&A] [answered]
Hello. Apologies for this naive question. Is there a specific threshold that can be used to determine whether the accessibility visualized through CoveragePlot could be of significance/biological relevance. 
For instance,
-Scenario 1:  for gene A, there is a small peak in Macrophages in Treated and no peak in Control with Normalized signal range of 0-8.5
-Scenario 2: for gene B, if there is a prominent peak in Macrophages in Treated and no peak in Control with Normalized signal range of 0-46

For gene B the difference is clear enough/signal is high that we could consider that the accessibility may potentially correlate with some biological signal
For gene A, does it make sense to still consider the difference in accessibility may be due to biology, or is the value too low? Could sparsity account for the absence of signal in the Control?
The small peak for gene A does not show up in the results from differential accessibility analysis on peaks, while the peak for gene B does show up during differential analysis.

Any guidance/insights regarding this query would be immensely helpful.

I am grateful for all your help and support. Thank you so much

**Comment by timoast** [comment, marked as answer]
> Is there a specific threshold that can be used to determine whether the accessibility visualized through CoveragePlot could be of significance/biological relevance.

There's no specific threshold

It's difficult to give specific advice without knowing anything about the expected biology or question being studied. If the differences are small it can be a good idea to try to replicate the results.

**Reply by jarchana09** [reply]
Thank you very much Tim for your inputs on the threshold. Really appreciate your insights.
